packages/core/src/indexing-store/historicalStore.ts

```diff
@@ -112,7 +112,7 @@ export const getHistoricalIndexingStore = ({
       } else {
         updateObject = data;
       }
-      const updateRow = encodeRow(updateObject, table, kind);
+      const updateRow = encodeRow({ id, ...updateObject }, table, kind);
```

Comment:
For posterity - we have to include the `id` here to handle the case where `updateObject = {}`. Otherwise, it's SQL syntax error because the SET clause has no values in it.